zeppelin-plugins/pom.xml

```diff
@@ -25,10 +25,8 @@
         <artifactId>zeppelin</artifactId>
         <groupId>org.apache.zeppelin</groupId>
         <version>0.10.0-SNAPSHOT</version>
-        <relativePath>..</relativePath>
```

Comment:
I think `relativePath` should still be used at this point, but it should point to the pom.xml.
```
<relativePath>../pom.xml</relativePath>
```
I know that the parent directory is the default and can be omitted, but sometimes you don't run Maven over the whole project, like in our [tests](https://github.com/apache/zeppelin/blob/97ce203a7fb72fd0300ff84ea5fa8e52f6125ebc/.github/workflows/core.yml#L67), then we should reference the parent project with a relative path.

> However, that would work if the parent project was already installed in our local repository or was in that specific directory structure (parent pom.xml is one directory higher than that of the module's pom.xml).
https://maven.apache.org/guides/introduction/introduction-to-the-pom.html#Project_Inheritance

Reply:
The default value of the tag `relativePath` is "../pom.xml". Nevertheless, do you think it should be declared explicitly?
> The relative path of the parent <code>pom.xml</code> file within the check out. If not specified, it defaults to <code>../pom.xml</code>.
> https://maven.apache.org/xsd/maven-4.0.0.xsd

Reply:
You are right, I changed my mind because of the good xsd.
Defaults should not be declared, so you should remove this.